src/EnergyPlus/HeatingCoils.cc

```diff
@@ -1860,6 +1860,95 @@ namespace HeatingCoils {
         default:
             break;
         }
+
+        // std 229 heating coils existing table adding new variables:
```

Comment:
Why is this in here but commented?  I may need to read back through the PR to understand why.